### DISCUSSION TITLE
Geotiff format in mapsforge

### DISCUSSION THREAD

**Discussion by AFA141** [mapsforge/mapsforge#1348] [Q&A] [answered]
I am making application with more map formats, I am already using mapsforge format now i want to add Geotiff format.
Is there any way I can use geotiff format without library for geotiff maps.. I can create tiles from geotiff and then use it, but I want to use that geotiff map picture without converting to tiles. 
Platform: Desktop
Thanks

**Comment by devemux86** [comment, marked as answer]
Map tiles are needed to use images as maps.
